extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -88,9 +88,10 @@ export class AmpAdXOriginIframeHandler {
    * Sets up listeners and iframe state for iframe containing ad creative.
    * @param {!Element} iframe
    * @param {boolean=} opt_isA4A when true do not listen to ad response
+   * @param {boolean=} opt_isRefreshing when true, leave ad loader up
    * @return {!Promise} awaiting render complete promise
    */
-  init(iframe, opt_isA4A) {
+  init(iframe, opt_isA4A, opt_isRefreshing) {
```

Comment:
Let's discuss offline but ideally we wouldn't polluate xorigin handler for this case